app/views/cases/show.html.slim

```diff
@@ -80,15 +80,15 @@ div id="case-#{@correspondence_type_key}" class="case"
     section.case-info
       = render partial: 'cases/case_attachments', locals: {case_details: @case}
 
-  - if !@case.offender_sar? && !@case.offender_sar_complaint?
+  - if !@case.offender_sar?
     section#messages-section.case-info
       = render partial: 'cases/case_messages', locals: { case_details: @case }
 
   - if @case_transitions.any?
     section.case-info.has-page-break
       = render partial: 'cases/case_history', locals: { case_transitions: @case_transitions }
 
-  - if @case.offender_sar? || @case.offender_sar_complaint?
+  - if @case.offender_sar?
```

Comment:
@Polsonby ,  I am wondering this line, if the offender_sar still means standard one not complaint one, you may need to change this flag to the new flag e.g. `offender_sar_any?` otherwise, it may not work for complaint case?